test/03_reduction/reduction_test.cpp

```diff
@@ -94,19 +95,21 @@ namespace hiptensor
         auto beta       = std::get<5>(param);
         auto op         = std::get<6>(param);
 
-        EXPECT_TRUE((lengths.size() > 1) && (lengths.size() <= 6));
-        EXPECT_TRUE((outputDims.size() >= 1) && (outputDims.size() <= 6));
+        EXPECT_TRUE((lengths.size() > 0) && (lengths.size() <= 6));
+        EXPECT_TRUE((outputDims.size() >= 0) && (outputDims.size() < 6));
```

Comment:
For the case where outputDims == 6, possible that the operation is simply a permutation?
Should we consider this case?